plugins/inputs/win_perf_counters/README.md

```diff
@@ -38,6 +38,12 @@ It is recommended NOT to use this on OSes starting with Vista and newer because
 Example for Windows Server 2003, this would be set to true:
 `PreVistaSupport=true`
 
+#### UseWinTimestamps
```

Comment:
Maybe a more descriptive name would be `UsePerfCounterTime`?

Reply:
Yup. I like it. Changed